actionpack/lib/action_controller/metal/rescue.rb

```diff
@@ -8,6 +7,16 @@ module Rescue
     extend ActiveSupport::Concern
     include ActiveSupport::Rescuable
 
+    def rescue_with_handler(exception)
+      if ActionView::Template::Error === exception \
```

Comment:
Action Controller should not know anything about Action View. This code should be in action view